homeassistant/components/comelit/coordinator.py

```diff
@@ -78,3 +85,76 @@ async def _async_update_data(self) -> dict[str, Any]:
             raise UpdateFailed(f"Error fetching data: {repr(err)}") from err
         except exceptions.CannotAuthenticate:
             raise ConfigEntryAuthFailed
+
+
+class ComelitVedoSystem(DataUpdateCoordinator):
```

Comment:
These 2 coordinators really look a lot like each other. Can't you maybe create 1 superclass with all the logic and then 2 subclasses which implement the update data function

Reply:
Done